HelloQuad/HelloQuad.cs

```diff
@@ -0,0 +1,148 @@
+using AtomicEngine;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        Application.Run<HelloQuad>(args);
+    }
+}
+
+public class HelloQuad : AppDelegate
+{
+    // Scene reference kept here so it won't be collected by the GC
+    Scene scene;
+    Camera camera;
+    Graphics graphics;
+    Viewport viewport;
+
+    Texture2D texture;
+    VertexBuffer vertexBuffer;
+
+    public override void Start()
+    {
+        Renderer renderer = AtomicNET.GetSubsystem<Renderer>();
+        graphics = AtomicNET.GetSubsystem<Graphics>();
+        viewport = renderer.GetViewport(0);
+        
+        scene = new Scene();
+        scene.CreateComponent<Octree>();
+        viewport.Scene = scene;
+
+        camera = scene.CreateChild("Camera").CreateComponent<Camera>();
+        camera.Node.Position = new Vector3(0.5f, 0.5f, 0.0f);
+        camera.Orthographic = true;
+        camera.OrthoSize = 1.5f;
+        viewport.Camera = camera;
+
+        // We create a XML from string so this code is fully self-contained
+        XMLFile xml = new XMLFile();
+        xml.FromString("<renderpath><command type=\"sendevent\"/></renderpath>");
+
```

Comment:
Can we tap into the view render end event instead of modifying render path? https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Graphics/GraphicsEvents.h#L94